File: GFG/graph/graph_coloring/leetcode_785.go
package main

import "math"

func colorGraph(graph [][]int, vtx, clr int, colors []int) {
	newColor := int(math.Abs((float64)(1 - clr)))
}
func isBipartite(graph [][]int) bool {
	V := len(graph)
	colors := make([]int, V)
	for i := range colors {
		colors[i] = -1
	}

	for i := 0; i < V; i++ {
		if colors[i] == -1 {

		}
	}
}

func main() {

}

File: GFG/graph/graph_coloring/leetcode_785.cpp
#include <bits/stdc++.h>
using namespace std;
typedef long long ll;

class Solution
{
public:
    void color_graph(vector<vector<int>> graph) bool isBipartite(vector<vector<int>> &graph)
    {
        int n = graph.size();
        vector<int> colors(n, -1);
        for (int i = 0; i < n; i++)
        {
            if (colors[i] == -1)
            {
                color_graph(graph, i, colors, 0)
            }
        }
    }
};